samples/drivers/led_lp5562/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _lp5562:
+
+LP5562: 4-Channel RGB
+######################
+
+Overview
+********
+
+This sample controls 4 LEDs connected to a TI LP5562 driver, using the
+following pattern:
+
+ 1. turn on LEDs to be red
+ 2. turn on LEDs to be green
+ 3. turn on LEDs to be blue
+ 4. turn on LEDs to be white
+ 5. turn on LEDs to be yellow
+ 6. turn on LEDs to be purple
+ 7. turn on LEDs to be cyan
+ 8. turn on LEDs to be orange
+ 9. turn off LEDs
+ 10. blink the LEDs in white
+ 11. turn off LEDs
+ 12. blink the LEDs in purple
+ 13. turn off LEDs
+
+The sample assumes that RGB LEDs are connected in the intended colors to the
+according channels. Refer to the manual linked below to find the mapping.
+
+Building and Running
+********************
+
+Build the application for the :ref:`nrf52840_pca10056` board, and connect
+a LP5562 LED driver on the bus I2C0 at the address 0x30.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/led_lp5562
+   :board: nrf52840_pca10056
+   :goals: build
+   :compact:
+
+For flashing the application, refer to the Flashing section of the
+:ref:`nrf52840_pca10056` board documentation.
+
+References
```

Comment:
I'd recommend deleting this heading and change the bullet to this and change the reference noted above:

    .. _LP5562 Manual: http://www.ti.com/lit/ds/symlink/lp5562.pdf

Reply:
done. Thanks for the feedback.